fapolicy_analyzer/glade/main_window.glade

```diff
@@ -176,6 +176,39 @@
                     </child>
                   </object>
                 </child>
+                <child>
+                  <object class="GtkMenuItem">
+                    <property name="visible">True</property>
+                    <property name="can-focus">False</property>
+                    <property name="label" translatable="yes">System</property>
+                    <child type="submenu">
+                      <object class="GtkMenu">
+                        <property name="visible">True</property>
+                        <property name="can-focus">False</property>
+                        <child>
+                          <object class="GtkImageMenuItem" id="fapdStopMenu">
```

Comment:
Can we add a submenu titled `fapolicyd` and then under it have star and stop options?

Reply:
Sure 'System' was SD's suggestion iirc or the result of a discussion between he and I; I think the intent at that time was to interact w/other externals under that menu. It is an easy change.

Reply:
The `System` menu is fine, I was suggesting a flow like `System -> fapolicyd -> Start / Stop`

Reply:
I'd prefer to push that change then, until be have other submenu items. I'd like to get this merged prior to that change.